### PR TITLE
ADDED: Emotes Purr, Trill, Meow, Growl to Felionoid + Audio

### PR DESCRIPTION
**Pull request by Cerzix**
## Short description

Adding additional sounds to emotes for Felionoids

## Checks

- [ ] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Cerzix
- add: Added Purr emote audio for felionoids.
- add: Added Trill emote audio for felionoids.
- add: Added Meow emote audio for felionoids.
- add: Added Growl emote audio for felionoids.

### CODE REVIEW

File: SpaceStation14.sln
Comment (fqqf) [inline SpaceStation14.sln:1]: xd (its fine though, dont bother)

Comment (Cerzix) [reply SpaceStation14.sln:1]: Was wondering about that, is that like auto added? Do i delete it next time? :D

Comment (fqqf) [reply SpaceStation14.sln:1]: aye, its a solution file. Next time just do ```git restore``` on it

Comment (Cerzix) [reply SpaceStation14.sln:1]: gotcha, will do!!

File: Resources/Audio/_Starlight/Voice/Felionoid/attributions.yml [hunk 1]
```diff
@@ -11,4 +11,23 @@
...
+- cat_meow6.ogg
+- cat_purr1.ogg
+- cat_purr2.ogg
+- cat_purr3.ogg
+- cat_trill1.ogg
+- cat_trill2.ogg
+- cat_trill3.ogg
+license: "CC0-1.0"
+copyright: "Taken from https://pixabay.com/"
+source: "https://github.com/ss14Starlight/space-station-14"
```
Comment (Rinary1) [inline Resources/Audio/_Starlight/Voice/Felionoid/attributions.yml]: space?

```suggestion
- files:
  - cat_growl1.ogg
  - cat_growl2.ogg
  - cat_meow1.ogg
  - cat_meow2.ogg
  - cat_meow3.ogg
  - cat_meow4.ogg
  - cat_meow5.ogg
  - cat_meow6.ogg
  - cat_purr1.ogg
  - cat_purr2.ogg
  - cat_purr3.ogg
  - cat_trill1.ogg
  - cat_trill2.ogg
  - cat_trill3.ogg
  license: "CC0-1.0"
  copyright: "Taken from https://pixabay.com/"
  source: "https://github.com/ss14Starlight/space-station-14"
```

File: SpaceStation14.sln [hunk 2]
```diff
@@ -485,8 +485,8 @@ Global
 		{07CA34A1-1D37-4771-A2E3-495A1044AE0B} = {7844DA69-B0F0-49FB-A05E-ECA37372277A}
 		{6FBF108E-5CB5-47DE-8D7E-B496ABA9E3E2} = {7844DA69-B0F0-49FB-A05E-ECA37372277A}
 		{83F510FE-9B50-4D96-AFAB-CC13998D6AFE} = {7844DA69-B0F0-49FB-A05E-ECA37372277A}
-		{5C05B9B4-6AFE-4884-AA6A-5A26C0FFF2F6} = {83B4CBBA-547A-42F0-A7CD-8A67D93196CE}
 		{7844DA69-B0F0-49FB-A05E-ECA37372277A} = {83B4CBBA-547A-42F0-A7CD-8A67D93196CE}
+		{5C05B9B4-6AFE-4884-AA6A-5A26C0FFF2F6} = {83B4CBBA-547A-42F0-A7CD-8A67D93196CE}
```
Comment (Rinary1) [inline SpaceStation14.sln:489]: ???

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: SpaceStation14.sln (hunk 2 above)
Comment (Cerzix) [reply SpaceStation14.sln:489]: > ???

was added automatically, see other convo, will remove it next time

**Pull request merged**